### PR TITLE
gripper_enablement

### PR DESCRIPTION
**Pull request by MiguelSolisSegura**
## Summary

This PR introduces gripper control and usage of `telekinesis_interfaces` ROS package.

## What Changed?

- Updated configuration files to include gripper open and close parameters
- Modified hand tracker node to detect pinching gestures and publish TelekinesisGoal messages
- Updated telekinesis goals node to handle TelekinesisGoal messages and incorporate grasp information
- Updated package dependencies to include telekinesis_interfaces
- Modified TelekinesisGoal message structure to use an int8 for grasp instead of a GraspMode
- Enhanced logic to avoid wasting computation resources in the  `hand_tracker_node.py`.

## Why is This Change Required?

These changes improve the robot control system by adding gripper functionality and enhancing the hand tracking capabilities. The new pinch detection allows for a slightly improved control of the robot's gripper, with respect to the previous raw implementation.

## How to Test?

Refer to the instructions provided in the README file at the `modular_droids` folder. Gripper values for the servo are already included for PX100 in the configuration files, but it is required to fill them for VX300.

### CODE REVIEW

**Comment by MiguelSolisSegura** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/droidsx/droid-bodies/36?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#36** <a href="https://app.graphite.dev/github/pr/droidsx/droid-bodies/36?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#35** <a href="https://app.graphite.dev/github/pr/droidsx/droid-bodies/35?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#34** <a href="https://app.graphite.dev/github/pr/droidsx/droid-bodies/34?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @MiguelSolisSegura and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by MichaelrMentele — APPROVED** [review]
LGTM

[shipshipship](https://app.graphite.dev//settings/meme-library?org=droidsx)

File: bodies/modular_droids/modular_droids/hand_tracker_node.py [hunk 1]
```diff
@@ -26,35 +27,64 @@ def on_device_event(self, event):
...
+        digit = hand.digits[digit_idx]
+        return digit.distal.next_joint
+
+    def sub_vectors(self, v1: ldt.Vector, v2: ldt.Vector):
+        return map(float.__sub__, v1, v2)
+
+    def fingers_pinching(self, thumb: ldt.Vector, index: ldt.Vector):
+        diff = list(map(abs, self.sub_vectors(thumb, index)))
+        norm = np.linalg.norm(np.array(diff))
+        pinching = int(5 / 3 * norm - 100 / 3)
```
Comment (MichaelrMentele) [inline bodies/modular_droids/modular_droids/hand_tracker_node.py:70]: good to leave a comment for numbers like this -- they seem random 👿

File: bodies/modular_droids/modular_droids/tentacol_controller_node.py [hunk 2]
```diff
@@ -24,6 +24,9 @@ def __init__(self):
         super().__init__("dynamixel_controller_node")
         self.get_logger().info("Initializing TentacolControllerNode...")
 
+        # Declare default values
+        self.gipper_default = -999.9
```
Comment (MichaelrMentele) [inline bodies/modular_droids/modular_droids/tentacol_controller_node.py:28]: The comment is useless right now, explain why -999.9?

Comment (MiguelSolisSegura) [reply bodies/modular_droids/modular_droids/tentacol_controller_node.py:28]: Totally, I've added a more infromative comment. Also I would like to create a wiki including explanations for all the parameters in our config files.

File: bodies/modular_droids/modular_droids/hand_tracker_node.py (hunk 1 above)
Comment (MiguelSolisSegura) [reply bodies/modular_droids/modular_droids/hand_tracker_node.py]: Yes, this has been fixed in latest version.

**Comment by MiguelSolisSegura** [comment]
### Merge activity

* **Sep 30, 4:57 PM EDT**: @MiguelSolisSegura started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/droidsx/droid-bodies/36).

**Pull request merged**